python/ray/_private/conda.py

```diff
@@ -1,11 +1,14 @@
 import logging
```

Comment:
Really minor but can we make a subdir like `ray/_private/environments` and put the files in there? Can do it in a separate/future PR

Reply:
I'll do it in a future PR, I made an issue here: https://github.com/ray-project/ray/issues/15890